src/main/java/com/mongodb/jdbc/oidc/JdbcOidcCallbackContext.java

```diff
@@ -16,24 +16,28 @@
 
 package com.mongodb.jdbc.oidc;
 
+import com.mongodb.MongoCredential.IdpInfo;
+import com.mongodb.MongoCredential.OidcCallbackContext;
 import java.time.Duration;
 
-// TODO: This class is a placeholder for the OidcCallbackContext,
-//       it will be removed when Java Driver OIDC support is added.
-public class OidcCallbackContext {
+public class JdbcOidcCallbackContext implements OidcCallbackContext {
     private Duration timeout;
     private int version;
     private String refreshToken;
     private IdpInfo idpInfo;
 
-    public OidcCallbackContext(
+    public JdbcOidcCallbackContext(
             Duration timeout, int version, String refreshToken, IdpInfo idpInfo) {
         this.timeout = timeout;
         this.version = version;
         this.refreshToken = refreshToken;
         this.idpInfo = idpInfo;
     }
 
+    public String getUserName() {
```

Comment:
Should this be empty string or null? I'm not sure, but let's be certain. Oh, also, OIDC _can_ have user names even though our test server doesn't require them (because it gets them from the IdP).

Reply:
The interface annotation has @nullable, so Null seems better.
And I agree with Patrick, if providing a username is possible, we should should support that too. Especially since it looks like you allow for a username to be provided via the connection settings for OIDC and only reject password.

Reply:
Okay, I added the username in the constructor similar to the [java driver](https://github.com/mongodb/mongo-java-driver/blob/790185f9ed64fed4e5720e2bbbff170d247a7374/driver-core/src/main/com/mongodb/internal/connection/OidcAuthenticator.java#L660), too bad their class isn't public.  For us, this `JdbcOidcCallbackContext` class is only used in the manual OIDC tests.